doc/source/structures/orbits/orbit.rst

```diff
@@ -78,9 +103,15 @@ Structure
         * - :attr:`POSITION`
           - :struct:`Vector`
           - The current position
+        * - :attr:`POSITIONAT(time)`
+          - :struct:`Vector`
+          - The position at the given time
         * - :attr:`VELOCITY`
           - :struct:`Vector`
           - The current velocity
+        * - :attr:`VELOCITYAT(time)`
+          - :struct:`Vector`
+          - The velocity at the given time
         * - :attr:`NEXTPATCH`
           - :struct:`Orbit`
           - Next :struct:`Orbit`
```

Comment:
It occurs to me that if we're having suffix-based `:POSITIONAT` and `:VELOCITYAT` for `OBT` objects, we should probably be consistent and make them for `BODY` and `VESSEL` too.

The only reason the built-in functions POSITIONAT(thing,time) and VELOCITYAT(thing,time) are *not* suffixes is purely because they were implemented prior to the introduction of executable suffixes (methods) to the language.  Executable suffixes is definitely the more logical way to do them.  We can leave the built-in functions there as well for backward compatibility, and behind the scenes redirect both of them to execute the same chunk of code.